src/selectors/getChildren.ts

```diff
@@ -130,6 +130,56 @@ const getChildrenSortedAlphabetical = moize(
   },
 )
 
+/** Generates children sorted by their creation date. */
+const getChildrenSortedCreated = moize(
+  (state: State, id: ThoughtId): Thought[] => {
+    const sortPreference = getSortPreference(state, id)
+    const sorted = getChildrenSortedBy(state, id, (a, b) => {
+      return sortPreference.direction === 'Desc'
+        ? b.created > a.created
+          ? 1
+          : b.created < a.created
+            ? -1
+            : 0
+        : a.created > b.created
+          ? 1
+          : a.created < b.created
+            ? -1
+            : 0
```

Comment:
Did you mean to import `compareThoughtByCreated`? The low level comparison logic should probably go there.